docs/integrations/pydantic.md

```diff
@@ -264,3 +264,34 @@ input_data = UserInput(id='abc', name='Jake')
 # this will run pydantic's validation
 instance = input_data.to_pydantic()
 ```
+
+### Retaining input information
+
+When looking to use pydantic as a validation layer and export models afterwards,
+it is sometimes important to retain metadata about the original input. In strawberry,
+this is possible via the `with_input_metadata` flag on input models.
```

Comment:
Minor styling adjustment:
```suggestion
When looking to use pydantic as a validation layer and export models afterwards,
it is sometimes important to retain metadata about the original input. In Strawberry,
this is possible via the `with_input_metadata` flag on input models.
```

Reply:
I think this part needs to be rephrased though:
> When looking to use pydantic as a validation layer and export models afterwards

Perhaps
> When using pydantic as a validation layer, it is sometimes important to retain metadata about the original input while exporting models.

Reply:
Change reflected and pushed. Thank you for improving the wording!